src/reports/citytax/report.ts

```diff
@@ -0,0 +1,102 @@
+import {BerlinCityTaxRowItemModel, CityTax} from "./interfaces";
+import {addDays, alert, formattedExecutionTime, getDates, lodash} from "shared";
+import {getAccountTransactions, getReservations} from "api/data";
+import {BookingModels, FinanceModels} from "api/schema";
+
+
+export function generateCityTaxReport(city: string, property: string, startDate: string, endDate: string) {
+    const cityTax = CityTax[city as keyof typeof CityTax];
```

Comment:
let's get rid of type assetion

```typescript
export function generateCityTaxReport(city: keyof typeof CityTax, property: string, startDate: string, endDate: string) {
    const cityTax = CityTax[city];

```


or we can use type and string value instead:

```typescript
type CityTaxType = 'HAMBURG' | 'BERLIN'

export function generateCityTaxReport(city: CityTaxType, property: string, startDate: string, endDate: string) {
    if (city === 'HAMBURG') {
        alert('Hamburg city tax report will be implemented soon!')
        return;
    }
    const transactionStartDate = addDays(startDate, -7);
    const transactionEndDate = addDays(endDate, 7);
    const transactions = getTransactions(property, transactionStartDate, transactionEndDate, startDate, endDate);
    const reservations = getReservations(property, transactionStartDate, transactionEndDate);

    const sheet = createSheetWithReportInfo(city, property, endDate, startDate);

    switch (city) {
        case 'BERLIN':
            generateBerlinCityTax(sheet, transactions, reservations);
            break;
    }
}
```